docs/getting-started/cash-app-pay.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: default
+title: Cash App Pay
+parent: Getting Started
+nav_order: 4
+---
+
+# Cash App Pay
+{: .d-inline-block .no_toc }
+NEW (v4.3.0)
+{: .label .label-green }
+
+
+<details markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+
+{: .alert }
+> Cash App Pay is currently available in the following region(s): US
+
+Cash App Pay support can be added using your Afterpay merchant account. This requires a token to be generated by a server-to-server call to the Afterpay API Create Checkout endpoint with a parameter of `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App PayKit SDK.
+
+{: .info }
+> When creating a checkout token, both `redirectConfirmUrl` and `redirectCancelUrl` must be set. Failing to do so will cause undefined behavior. The SDK’s example merchant server sets the parameters [here][example-server-props]{:target='_blank'}. See more by checking the [api reference][api-reference-props]{:target='_blank'}.
+
+## Step 1: Import the Cash App PayKit Dependency
+
+You can get the the latest version of the SDK from Maven. This is the import definition using Gradle:
+
+```gradle
+implementation "app.cash.paykit:core:1.0.3"
+```
+
+For definitions of other build systems, see [Cash PayKit on Maven Central][cash-on-maven]{:target="_blank"}.
+
+{: .info }
+> Version `v1.0.3` of the SDK size is `12.3 kB`.
+
+## Step 2: Create a Cash App PayKit SDK Instance
+
+To create a new instance of the Cash App PayKit SDK, you must pass the `clientId`. This is a required field. This can be retrieved through the Afterpay object: `Afterpay.environment.payKitId`.
```

Comment:
I wonder if it would make sense for us to rename `payKitId` to `payKitClientId`, for having a better 1-1 correspondence. WDYT?

Reply:
Changed for docs in [a0b7a0f](https://github.com/afterpay/sdk-android/pull/185/commits/a0b7a0f60c793dc140b969b286d60c7951eb721d). Will update code in #184